packages/external-interfaces/README.md

```diff
@@ -0,0 +1,10 @@
+# @smithy/external-interfaces
+
+[![NPM version](https://img.shields.io/npm/v/@smithy/external-interfaces/latest.svg)](https://www.npmjs.com/package/@smithy/external-interfaces)
+[![NPM downloads](https://img.shields.io/npm/dm/@smithy/external-interfaces.svg)](https://www.npmjs.com/package/@smithy/external-interfaces)
+
+This package provides a control for external interfaces.
+This package is installed to generated clients with the `^` version range, allowing updates
+of this package to deploy patches of external packages without publishing incremental versions of those packages.
+
+It is a hot patch-in-place mechanism. 
```

Comment:
- [ ] add a remark here that any breaking api change to how external APIs within this package are used must involve a major version bump of `@smithy/external-interfaces` because it is generated as `^1.0.0` to the clients